app/src/main/res/layout/activity_firebase_uiactivity.xml

```diff
@@ -3,7 +3,6 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".SignInActivity">
-
+    android:layout_height="match_parent">
+<!--    tools:context=".SignInActivity"> -->
```

Comment:
nit: I don't believe this needs to be commented out (even though it was red). This is generated by firebase I believe.

Reply:
Done :)